redwood/src/lib.rs

```diff
@@ -124,7 +124,10 @@ pub fn is_valid_secret_key(input: &str, passphrase: String) -> Result<String> {
 /// Encrypt a message (text) for the specified recipients. The list of
 /// recipients is a set of PGP public keys. The encrypted message will
 /// be written to `destination`.
-#[pyfunction]
+///
+/// This macro signature addresses a new pyo3 warning about default values...
```

Comment:
I'm going to move this comment to the commit message since it describes why the change was made, but doesn't need to be kept in the code itself.